src/plugins/vis_type_timelion/server/series_functions/es/lib/create_date_agg.js

```diff
@@ -29,29 +29,41 @@ export default function createDateAgg(config, tlConfig, scriptedFields) {
   };
 
   dateAgg.time_buckets.aggs = {};
-  _.each(config.metric, function (metric) {
-    metric = metric.split(':');
-    if (metric[0] === 'count') {
+  (config.metric || []).forEach((metric) => {
+    const metricBody = {};
+    const [metricName, metricArgs] = metric.split(/:(.+)/);
+    console.log('METRIC_TYPES.COUNT', METRIC_TYPES.COUNT);
+    console.log('METRIC_TYPES.PERCENTILES', METRIC_TYPES.PERCENTILES);
```

Comment:
@Dmitriynj please remove the console.logs :D

Reply:
Sorry, done